docs/sources/installation/configuration.md

```diff
@@ -147,7 +147,7 @@ Another way is put a webserver like Nginx or Apache in front of Grafana and have
 
 `http`,`https`,`h2` or `socket`
 
-> **Note** Grafana versions earlier than 3.0 are vulnerable to [POODLE](https://en.wikipedia.org/wiki/POODLE). So we strongly recommend to upgrade to 3.x or use a reverse proxy for ssl termination.
+> **Note.** Grafana versions earlier than 3.0 are vulnerable to [POODLE](https://en.wikipedia.org/wiki/POODLE). So we strongly recommend to upgrade to 3.x or use a reverse proxy for ssl termination.
```

Comment:
What's the reasoning behind adding the period? We don't do this anywhere else in our docs either.

Reply:
it is there in the previous Note section, just makes the look and feel of the document little better

Reply:
Oh yeah, I see there are few now (made a mistake when searching earlier). Personally I don't see the point of adding the period. What do you think @oddlittlebird?

Reply:
Instead of a period, use a colon `:` 

**Note:**

Reply:
Used colon , instead of period. Please merge the changes.